tests/functional/centos/7/cluster/hosts

```diff
@@ -1,6 +1,6 @@
 [mons]
 mon0 monitor_address=192.168.1.10
-mon1 monitor_interface=eth1
+mon1 monitor_address=192.168.1.11
```

Comment:
@guits The CI use case here was that we had a bug in the ceph.conf template that would cause the template to incorrectly render if the user had defined one mon host to use an address and the next in the list to use an interface. Or, if the mons had differing interfaces.  I think we need to keep a mix of these in the inventory for the tests that have 3 mons.